frontend/views/components/modal/Modal.vue

```diff
@@ -39,7 +42,7 @@ export default {
       if (to.query.modal) {
         // We reset the modals with no animation for simplicity
         if (to.query.modal !== this.content) this.content = to.query.modal
-        const subcontent = to.query.subcontent
+        const subcontent = to.query.subcontent ? to.query.subcontent.split('+').pop() : []
```

Comment:
Now that this PR supports `+`, does the modal still need `subcontent` variable, or can it do the same thing using just the `modal` variable? E.g. `?modal=Modal1+Modal2+Modal3` ?

Reply:
We need subcontent because there is two kind of modals with different behavior according to the modal type

Reply:
Ah, OK, makes sense 👍